<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_OpenFile"/>
   <addaction name="action_SaveFile"/>
   <addaction name="action_SaveSaveFile"/>
   <addaction name="separator"/>
   <addaction name="action_OpenQTextStream"/>
   <addaction name="action_SaveQTextStream"/>
   <addaction name="separator"/>
   <addaction name="action_Close"/>
  </widget>
  <action name="action_OpenFile">
   <property name="text">
    <string>QFile打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_SaveFile">
   <property name="text">
    <string>QFile另存为</string>
   </property>
   <property name="toolTip">
    <string>文件另存为</string>
   </property>
  </action>
  <action name="action_SaveSaveFile">
   <property name="text">
    <string>QSaveFile另存</string>
   </property>
   <property name="toolTip">
    <string>Save另存为</string>
   </property>
  </action>
  <action name="action_OpenQTextStream">
   <property name="text">
    <string>QTextStream打开</string>
   </property>
   <property name="toolTip">
    <string>Steam模式打开文件</string>
   </property>
  </action>
  <action name="action_SaveQTextStream">
   <property name="text">
    <string>QTextStream另存</string>
   </property>
   <property name="toolTip">
    <string>Stream模式文件另存为</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
